src/components/Dialog/Generic.tsx

```diff
@@ -38,9 +38,9 @@ const useActionButtonStyles = makeStyles(theme => ({
   mobileDialogActionsBox: {
     display: "flex",
     position: "fixed",
-    left: 0,
+    left: 8,
```

Comment:
@Marcel-Ebert Can you remember why we made it 8px left and right, instead of 0 as before?

Reply:
To make it smaller

Reply:
Yeah, but why? 😅
Can't remember anymore...

Reply:
Because you were unhappy with the space between the button(s) and the edge of the screen 🤐